src/main/groovy/edu/oregonstate/mist/invencycle/resources/BikeResource.groovy

```diff
@@ -93,6 +94,40 @@ class BikeResource extends Resource {
     }
 
     /**
+     * PUT bike with ID
+     */
```

Comment:
I would add some comments to your method to specify that you are only updating fields if specified.